CODEOWNERS

```diff
@@ -40,6 +40,7 @@ open-metadata-implementation/access-services/governance-engine/ @danielaotelea
 open-metadata-implementation/access-services/subject-area/ @davidradl
 open-metadata-implementation/access-services/asset-lineage/ @GuyAckermans @DimitriosMaimaris
 open-metadata-implementation/governance-servers/open-lineage-services/ @GuyAckermans @DimitriosMaimaris
+open-metadata-implementation/adapters/open-connectors/governance-daemon-connectors/open-lineage-connectors @GuyAckermans @DimitriosMaimaris
```

Comment:
According to the docs, the directory name needs a leading / - it may work without, but perhaps you could update just in case (see https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax)